File: Python/Program/Komponen.py
class Komponen:
    def __init__(self, nama="", harga=0, merk=""):
        self.nama = nama
        self.harga = harga
        self.merk = merk
    
    def set_nama(self, nama):
        self.nama = nama
    
    def set_harga(self, harga):
        self.harga = harga
    
    def set_merk(self, merk):
        self.merk = merk
    
    def get_nama(self):
        return self.nama
    
    def get_harga(self):
        return self.harga
    
    def get_merk(self):
        return self.merk
    
    def get_info(self):
        print(f"Nama: {self.nama}, Harga: {self.harga}, Merk: {self.merk}")

File: Java/Program/Komponen.java
public class Komponen {
    private String nama;
    private float harga;
    private String merk;

    public Komponen() {
    }

    public Komponen(String nama, float harga, String merk) {
        this.nama = nama;
        this.harga = harga;
        this.merk = merk;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setHarga(float harga) {
        this.harga = harga;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public String getNama() {
        return nama;
    }

    public float getHarga() {
        return harga;
    }

    public String getMerk() {
        return merk;
    }

    public void getInfo() {
        System.out.println("Nama: " + nama + ", Harga: " + harga + ", Merk: " + merk);
    }
}